services/121-service/src/payments/fsp-integration/commercial-bank-ethiopia/commercial-bank-ethiopia.service.ts

```diff
@@ -73,7 +73,6 @@ export class CommercialBankEthiopiaService
     programId: number,
     paymentNr: number,
   ): Promise<FspTransactionResultDto> {
-    // ##TODO: Regactor this to ensure that the programFinancialServiceProviderConfigurationId is the same for the whole list
```

Comment:
I think this todo was something I added because I first moved querying the cbe username and password out of the job. However later on I reversed this so this comment is outdated